### PR TITLE
configshell: export __version__ variable

### PR DESCRIPTION
**Pull request by pkalever**
$ python -c "import configshell_fb; print(configshell_fb.\_\_version\_\_)"
1.1.25

Signed-off-by: Prasanna Kumar Kalever <prasanna.kalever@redhat.com>

### CODE REVIEW

**Comment by pkalever** [comment]
@maurizio-lombardi @lxbsz please reveiw. Thanks!

File: configshell/__init__.py [hunk 1]
```diff
@@ -25,3 +25,8 @@
 from .node import ConfigNode, ExecutionError
 from .prefs import Prefs
 from .shell import ConfigShell
+
+__version__ = '1.1.25'
+__url__ = 'http://github.com/open-iscsi/configshell-fb'
+__description__ = 'A framework to implement simple but nice CLIs.',
```
Comment (lxbsz) [inline configshell/__init__.py]: Is ',' needed here ?

Comment (pkalever) [reply configshell/__init__.py]: good catch Xiubo, will fit it now. Thanks!

**Comment by pkalever** [comment]
@lxbsz updated. Thanks!

**Pull request merged**